paasta_tools/utils.py

```diff
@@ -97,7 +97,7 @@
 DEFAULT_SYNAPSE_HAPROXY_URL_FORMAT = "http://{host:s}:{port:d}/;csv;norefresh"
 
 DEFAULT_CPU_PERIOD = 100000
-DEFAULT_CPU_BURST_PCT = 900
+DEFAULT_CPU_BURST_ADD = 1
```

Comment:
This will cause a big bounce - you'll probably want to work with someone in Europe to ship this off-hours.

Reply:
Roger.